users/users.md

```diff
@@ -117,6 +118,80 @@ Response
 ```
 ---
 
+## Create user account
+
+Create a user account.
+
+```apacheconfig
+POST /v1/users
+```
+
+#### Path parameters
+The path parameters are not expected.
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+
+| Name | Type | Description | Required |
+| ---- | ---- | ------------| :---: |
+| email | `string` | The new user’s email. | &#9745; |
+| firstName | `string` | The new user’s first name. | &#9745; |
+| lastName | `string` | The new user’s last name. | &#9745; |
+| password | `string` | The new user’s password. | &#9745; |
```

Comment:
Please add in the description for the password what is the format of the password expected.